disperser/apiserver/server_v2.go

```diff
@@ -149,7 +154,36 @@ func (s *DispersalServerV2) GetBlobStatus(ctx context.Context, req *pb.BlobStatu
 }
 
 func (s *DispersalServerV2) GetBlobCommitment(ctx context.Context, req *pb.BlobCommitmentRequest) (*pb.BlobCommitmentReply, error) {
-	return &pb.BlobCommitmentReply{}, api.NewErrorUnimplemented()
+	if s.prover == nil {
+		return nil, api.NewErrorUnimplemented()
+	}
+	if len(req.GetData()) == 0 {
```

Comment:
Should we have bounds check on the largest supported blob size?

Reply:
good call